File: tempFiles/browserJavaToKotlin - Copy-old/src/main/java/com/luciasoft/browserjavatokotlin/multibrowser/MyYesNoDialog.java
package com.luciasoft.browser.multibrowser;

import android.content.Context;
import android.content.DialogInterface;

class MyYesNoDialog
{
	static void show(Context context, String title, String message, DialogInterface.OnClickListener onYesClick)
	{
		MyMessageBox.show(context, title, message, MyMessageBox.ButtonsType.YesNo, onYesClick, null);
	}
}

File: BrowserLib/src/main/java/com/luciasoft/browser/MyYesNoDialog.kt
package com.luciasoft.browser

import android.content.Context
import android.content.DialogInterface
import com.luciasoft.browser.MyMessageBox.Companion.show

internal object MyYesNoDialog
{
    fun show(
        context: Context,
        title: String = "",
        message: String = "",
        onYesClick: DialogInterface.OnClickListener? = null
    )
    {
        show(context, title, message, MyMessageBox.ButtonsType.YesNo, onYesClick, null)
    }
}